### PR TITLE
Returns only even numbers in an array

### PR DESCRIPTION
**Pull request by SimiS22**
Please review the code.

### CODE REVIEW

File: src/even-array/even-array.test.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { evenNumberArr } from './index'
+describe('outputd only the even numbers of an array', function () {
```
Comment (chin2km) [inline src/even-array/even-array.test.ts]: empty line after import

File: src/even-array/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
+export const evenNumberArr = (inputArr: number[]) => {
+    let outputArr: number[] = [];
+    for (let i = 0; i < inputArr.length; i++) {
+        let rem = inputArr[i] % 2;
+        if (rem == 0 && inputArr[i] != 0) {
+            true;
```
Comment (chin2km) [inline src/even-array/index.ts]: just true? ingane ezhuthiyal oru effectum illa.. 
its like writing:
5;
"simi";

in a line

File: src/even-array/index.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            outputArr.push(inputArr[i]);
+        }
+        else {
+            false;
+        }
+    }
+
+    return outputArr;
+
+}
```
Comment (chin2km) [inline src/even-array/index.ts]: also rewrite this using array.filter

Comment (SimiS22) [reply src/even-array/index.ts]: As suggested, code has been rewritten using array.filter().
Also, added the code for array filtering based on the element type.

File: .vscode/launch.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+            "skipFiles": [
+                "<node_internals>/**"
+            ],
+            "program": "${workspaceFolder}\\index.js",
+            "outFiles": [
+                "${workspaceFolder}/**/*.js"
+            ]
+        }
+    ]
+}
```
Comment (chin2km) [inline .vscode/launch.json]: delete this file.. not needed for now

File: src/even-array/index.ts [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+export const evenNumberArrUsingFilter = (inputArr: number[]) => {
+    function output(inValue: number): boolean {
+        return (inValue != 0 && inValue % 2 == 0)
+    }
+    return inputArr.filter(output);
+}
+
+
```
Comment (chin2km) [inline src/even-array/index.ts]: empty lines

**Pull request merged**